scripts/protein/esm2/esm2_pretrain.py

```diff
@@ -183,6 +183,7 @@ def main(
         strategy=strategy,
         limit_val_batches=limit_val_batches,  # This controls upsampling and downsampling
         val_check_interval=val_check_interval,
+        log_every_n_steps=log_every_n_steps,
```

Comment:
You need to add this in one other spot to get logging to be less frequent, in your `nl.MegatronStrategy` above add ```
        log_interval=log_every_n_steps,
```